Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -222,6 +222,7 @@
   - type: ItemBorgModule
     items:
     - SawAdvanced
+    - WeaponPistolEchis #Todo: Заменить на вариант Шприцемета с выбором снарядов
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)